auto_nag/scripts/defectenhancementtask.py

```diff
@@ -75,10 +85,32 @@ def get_bugs(self, date='today', bug_ids=[]):
                 'type': bug['type'],
                 'bugbug_type': suggestion,
                 'confidence': int(round(100 * prob[index])),
+                'autofixed': False,
             }
 
+            # Only autofix results for which we are sure enough.
+            # And only autofix defect -> task/enhancement for now, unless we're 100% sure.
+            if prob[index] >= self.get_config('confidence_threshold') and (bug['type'] == 'defect' or prob[index] == 1.0):
+                results[bug['id']]['autofixed'] = True
+                self.autofix_type[bug['id']] = suggestion
+
         return results
 
+    def get_autofix_change(self):
+        cc = self.get_config('cc')
+        return {
+            bug_id: {
+                'type': suggestion,
+                'cc': {
+                    'add': cc
+                },
+                'comment': {
+                    'body': 'The bot thinks this bug is a {}, but please change it back in case of error'.format(suggestion),
```

Comment:
nano-nit: add a point at the end of the sentence.

Reply:
Done with 3830239.